src/useVisibilityObserver/useVisibilityObserver.ts

```diff
@@ -0,0 +1,24 @@
+import { onMounted, ref, Ref } from 'vue'
+import { useIntersectionObserver } from '@/useIntersectionObserver'
+
+export function useVisibilityObserver(element: Ref<HTMLElement | undefined>): Ref<boolean> {
+  const visible = ref(false)
+
+  function intersect(entries: IntersectionObserverEntry[]): void {
+    entries.forEach(entry => {
+      if (entry.isIntersecting) {
+        visible.value = true
+        disconnect()
+      }
+    })
+  }
+
+  const { observe, disconnect } = useIntersectionObserver(intersect)
+
+  onMounted(() => {
+    observe(element)
+  })
+
+  return visible
```

Comment:
We should return an object of `{ visible, disconnect }` in case the dev wants to disconnect manually.